### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by ajb** [block-wallet/extension#304] [General]
Hi there,

I'm excited about this project! I've been looking for an open source browser extension wallet that I can customize, and hopefully contribute back to.

Would I (or any other community contributor) be able to get access to the packages necessary to build the code in this repo? I'm unclear how one is supposed to do that.

At the very least, I would like to be able to build this myself for security reasons.

**Comment by julianariel** [comment]
hey @ajb, thanks for your submission!

Starting today, access to private packages is no longer needed. We were working on removing some dependencies we had in private repos, so now you should be able to use the source code and build it locally. Please try again and let us know if you find any trouble.

Thanks!
